airbyte/constants.py

```
@@ -64,9 +65,8 @@ def _str_to_bool(value: str) -> bool:
     """Convert a string value of an environment values to a boolean value."""
     return bool(value) and value.lower() not in {"", "0", "false", "f", "no", "n", "off"}
 
-
-TEMP_DIR_OVERRIDE: Path | None = (
-    Path(os.environ["AIRBYTE_TEMP_DIR"]) if os.getenv("AIRBYTE_TEMP_DIR") else None
+TEMP_DIR: Path = Path(
```

Ruff (0.8.2)

Expected 2 blank lines after class or function definition, found (1)

Add missing blank line(s)

(E305)